sdb/commands/linux/per_cpu.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class LxPerCpuPtr(sdb.SingleInputCommand):
+    """
+    Return the per-CPU pointer for the given CPU.
+
+    EXAMPLES
+        Get the per-CPU data of the "kmalloc-8" kmem cache for CPUs 0 and 1:
+
+            sdb> slabs | filter obj.name == "kmalloc-8" | member cpu_slab
+            (struct kmem_cache_cpu *)0x2f040
+            sdb> slabs | filter obj.name == "kmalloc-8" | member cpu_slab | cpu_ptr 0
+            (struct kmem_cache_cpu *)0xffff944effc2f040
+            sdb> slabs | filter obj.name == "kmalloc-8" | member cpu_slab | cpu_ptr 1
+            (struct kmem_cache_cpu *)0xffff944effd2f040
+    """
+
+    names = ["cpu_ptr"]
```

Comment:
Since linux puts "percpu" in the name of everything that relates to these variables, I think it might make sense to include that in the name of the command as well, to make it easier to guess what the command does. If we went as far as removing the ability to specify which CPU, and just always output the pointers for all the CPUs (as suggested above), we could potentially just call the command `percpu`.

Reply:
Name changed. Thank you!